backend/private-graph/graph/schema.resolvers.go

```diff
@@ -926,6 +926,25 @@ func (r *mutationResolver) AddAdminToWorkspace(ctx context.Context, workspaceID
 	return adminID, nil
 }
 
+// DeleteInviteLinkFromWorkspace is the resolver for the deleteInviteLinkFromWorkspace field.
+func (r *mutationResolver) DeleteInviteLinkFromWorkspace(ctx context.Context, workspaceID int, workspaceInviteLinkID int) (bool, error) {
+	_, err := r.isAdminInWorkspace(ctx, workspaceID)
+	if err != nil {
+		return false, e.Wrap(err, "current admin is not in workspace")
+	}
+
+	if err := r.validateAdminRole(ctx, workspaceID); err != nil {
```

Comment:
Should only admins be able to delete invites? Seems like member users should not be able to delete other invites, but they can invite other members

Reply:
maybe it doesn't make sense for a member to invite another admin, only another member?

Reply:
I'll add an issue for this